src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -86,14 +87,14 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
     }
 
     /**
-     * POST /organizations/user/:userlogin/organization/:organizationId :
+     * POST organizations/:organizationId/users/:userlogin :
      * Add a user to an organization
      *
      * @param userLogin the login of the user to add
      * @param organizationId the id of the organization where the user should be added
      * @return empty ResponseEntity with status 200 (OK), or 404 (Not Found) otherwise
      */
-    @PostMapping("/organizations/user/{userLogin}/organization/{organizationId}")
+    @PostMapping("organizations/{organizationId}/users/{userLogin}") // TODO: should be organizations/{organizationId}/users/{userLogin}
```

Comment:
do you need TODO in here, since you already modified the mapping like that, im just asking 😊